package.json

```diff
@@ -9,7 +9,8 @@
     "generate": "nuxt generate",
     "lint": "npx eslint --ext .vue  --ext .js .",
     "prepare": "husky install",
-    "heroku-postbuild": "npm run build"
+    "heroku-postbuild": "npm run build",
+    "cypress:open": "cypress open"
```

Comment:
As per suggestion here: https://docs.cypress.io/guides/getting-started/opening-the-app

Cypress can be started via `npm run cypress:open` instead of writing the path to cypress in npm's `bin`